x-pack/platform/plugins/shared/stack_connectors/server/connector_types/openai/openai.ts

```diff
@@ -53,18 +56,64 @@ import {
   getRequestWithStreamOption,
   pipeStreamingResponse,
   sanitizeRequest,
+  validatePKICertificates,
 } from './lib/utils';
 
+function formatPEMContent(
+  pemContent: string,
+  type: 'CERTIFICATE' | 'PRIVATE KEY',
+  logger: Logger
+): string {
+  if (!pemContent) return pemContent;
```

Comment:
```suggestion
  if (!pemContent) return pemContent;
  if (pemContent.includes('\n') && pemContent.startsWith(`-----BEGIN ${type}-----`) && pemContent.endsWith(`-----END ${type}-----`)) {
  return pemContent;
}
```

Skip reformatting if already valid